test/rabbit_core_metrics_gc_SUITE.erl

```diff
@@ -329,3 +328,75 @@ x(Name) ->
     #resource{ virtual_host = <<"/">>,
                kind = exchange,
                name = Name }.
+
+%% -------------------------------------------------------------------
+%% Cluster Testcases.
+%% -------------------------------------------------------------------
+
+cluster_queue_metrics(Config) ->
+    VHost = <<"/">>,
+    QueueName = <<"cluster_queue_metrics">>,
+    PolicyName = <<"ha-policy-1">>,
+    PolicyPattern = <<".*">>,
+    PolicyAppliesTo = <<"queues">>,
+
+    Node0 = rabbit_ct_broker_helpers:get_node_config(Config, 0, nodename),
+    Node1 = rabbit_ct_broker_helpers:get_node_config(Config, 1, nodename),
+
+    % Note: must start these processes as there is a delay in starting them up
+    % when a node starts.
+    rabbit_ct_broker_helpers:rpc(Config, Node0, rabbit_core_metrics_gc, start_link, []),
+    rabbit_ct_broker_helpers:rpc(Config, Node1, rabbit_core_metrics_gc, start_link, []),
+    rabbit_ct_broker_helpers:rpc(Config, Node0, rabbit_mgmt_gc, start_link, []),
```

Comment:
This part of the test (call `rabbit_mgmt_gc`) could never work on the `rabbit` app, as the management application is not started. 
```

=== Ended at 2017-07-03 14:07:12
=== Location: [{rabbit_ct_broker_helpers,rpc,967},
              {rabbit_core_metrics_gc_SUITE,cluster_queue_metrics,383},
              {test_server,ts_tc,1533},
              {test_server,run_test_case_eval1,1053},
              {test_server,run_test_case_eval,985}]
=== Reason: undefined function rabbit_mgmt_gc:run_gc_soon/1
  in function  rpc:'-handle_call_call/6-fun-0-'/5 (rpc.erl, line 187)
```
```
=ERROR REPORT==== 3-Jul-2017::14:07:12 ===
Error in process <0.631.0> on node 'rmq-ct-cluster_tests-1-21000@localhost' with exit value:
{undef,[{rabbit_mgmt_gc,run_gc_soon,[],[]},
        {rabbit_mirror_queue_sync,syncer_loop,3,
                                  [{file,"src/rabbit_mirror_queue_sync.erl"},
                                   {line,266}]}]}
```